features/select_projects_metrics_based_on_preference.feature

```diff
@@ -38,24 +38,24 @@ Scenario: select preferred projects
 
 Scenario: select preferred metrics
 	When I am on the projects page
-	Then I should see "2.9" within "#LocalSupport_code_climate_metric"
-	And I should see "0.6" within "#WebsiteOne_github_metric"
+	Then I should see "2.9" within "#project_1_code_climate_metric"
```

Comment:
right although it assumes that you know the ids of the project - I'd prefer that we sanitized the project names myself.  Particularly in the cucumber test itself it's important to be able to see the link between the fixture data of a particular name and the name in the step - we could make a step to translate to id under the hood of course, but even when debugging the CSS in the browser I'd prefer to see the project name - i.e. it is more meaningful to humans ...